modules/utxo-lib/src/payments/p2tr.ts

```diff
@@ -41,7 +56,7 @@ export function p2tr(a: Payment, opts?: PaymentOpts): Payment {
       pubkey: typef.maybe(ecc.isXOnlyPoint),
       // the pub key(s) used for keypath signing.
       // aggregated with MuSig2* if > 1
-      pubkeys: typef.maybe(typef.arrayOf(ecc.isXOnlyPoint)),
+      pubkeys: typef.maybe(typef.arrayOf(typef.anyOf(ecc.isXOnlyPoint, isOrdinaryPubkey))),
```

Comment:
mixing and matching doesn't seem right
```suggestion
      pubkeys: typef.maybe(typef.anyOf(typef.arrayOf(ecc.isXOnlyPoint), typef.arrayOf(isOrdinaryPubkey))),
```